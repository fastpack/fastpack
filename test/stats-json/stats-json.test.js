const { fpack } = require('../helpers.js');

process.chdir(__dirname);

test('--stats=json add modulesPaths in prod', async () => {
  expect(
    JSON.parse(
      await fpack({
        input: 'index.js',
        output: 'bundle-1',
        stats: 'json'
      })
    )
  ).toMatchSnapshot();
});

test('--stats=json add modulesPaths in dev', async () => {
  expect(
    JSON.parse(
      await fpack({
        input: 'index.js',
        output: 'bundle-2',
        development: true,
        stats: 'json'
      })
    )
  ).toMatchSnapshot();
});
