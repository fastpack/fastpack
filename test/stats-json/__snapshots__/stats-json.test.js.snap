// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`--stats=json add modulesPaths in dev 1`] = `
Object {
  "modulesPaths": Array [
    "./dep1.js",
    "./dep2.js",
    "./index.js",
  ],
}
`;

exports[`--stats=json add modulesPaths in prod 1`] = `
Object {
  "modulesPaths": Array [
    "./dep1.js",
    "./dep2.js",
    "./index.js",
  ],
}
`;
